resources/api-gateway/README.md

```diff
@@ -0,0 +1,10 @@
+# API-Gateway
+
+## Overview
+API-Gateway is a component allowing exposure of a service through the kyma Console. It deploys and manages Istio and Ory/Oathkeeper CRDs.
+
+This chart installs a controller, which requires and uses the following applications/CRD, which should be installed beforehand:
```

Comment:
```suggestion
This chart installs the controller, which requires these CRDs to expose services:
```
a -> the - this chart installs a specific controller, not just any random controller ;)
removed "applications" - I see only CRDs listed here